lib/tire/model/indexing.rb

```diff
@@ -119,6 +171,22 @@ def create_elasticsearch_index
           false
         end
 
+        def log_alias_create(alias_name, target, response)
+          if response && response.code == 200
+            STDOUT.puts "Created an alias named '#{alias_name}' that points to '#{target}'"
```

Comment:
How about for non-error messages use the Configuration.logger.write "" method instead of STDOUT?